Mathlib/Topology/Algebra/Module/WeakBilin.lean

```diff
@@ -34,7 +34,7 @@ We prove the following results characterizing the weak topology:
 * `tendsto_iff_forall_eval_tendsto`: Convergence in `WeakBilin B` can be characterized
   in terms of convergence of the evaluations at all points `y : F`.
 
-## Notations
+## Notation
 
 No new notation is introduced.
```

Comment:
In fact, such sections should perhaps be deleted: feel free to make that a preliminary (or follow-up PR).